.github/workflows/lint.yml

```diff
@@ -32,4 +32,35 @@ jobs:
           if [[ "${{ steps.analysis.outputs.total }}" != "110" ]]; then
             echo "Expected score 110, got ${{ steps.analysis.outputs.total }}, please check the analysis report and resolve the issues"
             exit 1
-          fi
\ No newline at end of file
+          fi
+
+  coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: subosito/flutter-action@v1
+      - uses: actions/checkout@v2
+      - name: Generage test coverage
+        working-directory: objectbox
+        run: |
+          sudo apt-get install lcov
+          make depend
+          make coverage
+      - uses: devmasx/coverage-check-action@v1.2.0
+        with:
+          type: lcov
+          result_path: objectbox/coverage/lcov.info
+          min_coverage: 90
+          token: ${{ github.token }}
+      - uses: actions/setup-dotnet@v1
```

Comment:
https://github.com/actions/virtual-environments/blob/ubuntu20/20210614.1/images/linux/Ubuntu2004-README.md#net-core-sdk says the latest .NET Core SDK is always installed. No? Edit: k, I guess just copy/pasted from ReportGenerator.

Reply:
seems to work after removing, thx for the suggestion